microhapdb/marker.py

```diff
@@ -446,6 +446,16 @@ class Locus:
     def __init__(self, markers=None):
         self.markers = list() if markers is None else markers
 
+    @property
+    def definition(self):
+        variants = list()
+        for marker in sorted(self.markers, key=lambda m: m.name):
+            start, end = self.target_interval
+            offsets = [o - start for o in marker.offsets]
+            for offset, refr_offset in zip(offsets, marker.offsets):
+                variants.append((marker.name, offset, marker.chrom, refr_offset))
+        return pd.DataFrame(variants, columns=["Marker", "Offset", "Chrom", "ChromOffset"])
```

Comment:
Add a `locus.definition` attribute to grab all of the marker definitions for a locus with a shared start position.